content/python/concepts/built-in-functions/terms/locals/locals.md

```diff
@@ -0,0 +1,50 @@
+---
+
+Title: 'locals()'
+Description: 'The `locals()` function returns a dictionary with all the data necessary to run the program.'
+Subjects: 
+- 'Devloper tools'
+- 'Computer science'
+Tags:
+- 'Dictionary' 
+- 'Metadata' 
+- 'Symbol'
+Catalog-content: 
+- 'learn python 3'
+- 'paths/computer-science'
+
+---
+
+The `locals()`function returns a dictionary with all the data necessary to run the program.
+
+## Syntax
+
+```py
+locals()
+```
+
+## Example
+
+```py
+{'In': ['', 'locals()'],
+ 'Out': {},
+ '_': '',
+ '__': '',
+ '___': '',
+ '__builtin__': ,
+ '__builtins__': ,
+ '__name__': '__main__',
+ '_dh': ['/home/repl'],
+ '_i': '',
+ '_i1': 'locals()',
+ '_ih': ['', 'locals()'],
+ '_ii': '',
+ '_iii': '',
+ '_oh': {},
+ '_sh': ,
+ 'exit': ,
+ 'get_ipython': >,
+ 'quit': }
+```
+
+Attempting to edit an active variable by changing `locals()` does not work.
```

Comment:
Please add a ##Codebyte Example block at the end that will have the code for the term, the output showcasing does not follow it, just the code is fine.

Reply:
> I was trying to bring up the only other use for locals() that I know of. Did I do it wrong?

No no, you just need to add one more code under a ##Codebyte Example block

Reply:
Ah. I just need to show an example of that then. Got it. Thanks.